src/solver_cvodes.cpp

```diff
@@ -976,6 +1035,60 @@ static int fqBdot(realtype t, N_Vector x, N_Vector xB, N_Vector qBdot,
 }
 
 
+/**
+ * @brief Right hand side of differential equation for adjoint state xB
+ * when simulating in steadystate mode
+ * @param t timepoint
+ * @param xB Vector with the adjoint states
+ * @param xBdot Vector with the adjoint right hand side
+ * @param user_data object with user input @type Model_ODE
+ * @return status flag indicating successful execution
+ */
+static int fxBdot_ss(realtype t, N_Vector xB, N_Vector xBdot,
+                     void *user_data) {
+    auto model = static_cast<Model_ODE *>(user_data);
+    model->fxBdot_ss(t, xB, xBdot);
```

Comment:
Not sure how, but maybe there's another way to implement this, which does not operate on the model Jacobian... Opinions?